src/transaction/log_manager.c

```diff
@@ -14137,47 +14137,59 @@ cdc_get_start_point_from_file (THREAD_ENTRY * thread_p, int arv_num, LOG_LSA * r
     }
   else
     {
-      LOG_ARCHIVE_CS_ENTER (thread_p);
-      aligned_hdr_pgbuf = PTR_ALIGN (hdr_pgbuf, MAX_ALIGNMENT);
+      if (log_Gl.archive.vdes != NULL_VOLDES && log_Gl.archive.hdr.arv_num == arv_num)
+	{
+	  /* if target archive log volume is currenty mounted, then use that */
+	  process_lsa.pageid = log_Gl.archive.hdr.fpageid;
+	  process_lsa.offset = 0;
+	}
+      else
+	{
+	  LOG_ARCHIVE_CS_ENTER (thread_p);
```

Comment:
Not related with this issue, but there is no `LOG_ARCHIVE_CS_EXIT (thread_p);` when `fileio_is_volume_exist (arv_name)` is not true.

Reply:
Thank you, I will look into it

Reply:
I moved the LOG_ARCHIVE_CS_EXIT()